### PR TITLE
Task 320244: Move the dashboard parameter filter in widget embedding sample into github

### PR DESCRIPTION
**Pull request by naveena912**

### CODE REVIEW

File: Scenario Based Samples/Dashboard-Parameter-In-Widget-Embedding/README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+      ![initial-widget-rendering](https://github.com/boldbi/samples/assets/92368448/14b07a07-0f18-42e8-a130-f85a092ffffa)
+
+>**Note:** For more information about the initial rendering of the widget, please refer the [link](https://help.boldbi.com/embedding-options/embedding-sdk/embedding-a-widget/).
+
+6.  To modify the widget according to a particular dashboard parameter value, choose a different value from the list of `Departments`, then click on `Apply`. For example, select `Accounts` to demonstrate.
+
+    ![select-department-values](https://github.com/boldbi/samples/assets/92368448/6003e0dc-216f-4654-9217-4ff76cac32d0)
+
+>**Note:** For more information about updating the widget with dashboard parameter, please refer the [link](https://help.boldbi.com/embedding-options/embedding-sdk/embedding-api-reference/methods/#updatefilters).
```
Comment (vinothkumar1732) [inline Scenario Based Samples/Dashboard-Parameter-In-Widget-Embedding/README.md:93]: For more information about updating the widget with dashboard parameter without re-render the widget.

File: Scenario Based Samples/Dashboard-Parameter-In-Widget-Embedding/README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+     ![Embed Settings Download](https://github.com/boldbi/aspnet-core-sample/assets/91586758/d27d4cfc-6a3e-4c34-975e-f5f22dea6172)
+     ![EmbedConfig Properties](https://github.com/boldbi/aspnet-core-sample/assets/91586758/d6ce925a-0d4c-45d2-817e-24d6d59e0d63)
+
+  * Copy the downloaded `embedConfig.json` file and paste it into the designated [location](https://github.com/boldbi/aspnet-core-sample/tree/master/BoldBI.Embed.Sample) within the application. Please ensure you have placed it in the application, as shown in the following image.
+    
+    ![EmbedConfig image](https://github.com/boldbi/aspnet-core-sample/assets/91586758/bdb83a3e-02e4-4e99-ad57-717438e5ec5c)
+
+ ## Run a Sample Using Command Line Interface 
+
```
Comment (vinothkumar1732) [inline Scenario Based Samples/Dashboard-Parameter-In-Widget-Embedding/README.md:34]: Missed the point about to get the sample from github.

**Review by vinothkumar1732 — CHANGES_REQUESTED** [review]

File: Scenario Based Samples/Dashboard-Parameter-In-Widget-Embedding/README.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
+# Applying Dashboard Parameters in Widget Embedding BoldBI in .NET Core
```
[the remark this answers is not in the document]
Comment (naveena912) [reply Scenario Based Samples/Dashboard-Parameter-In-Widget-Embedding/README.md:1]: Resolved the concern and changed the content

File: Scenario Based Samples/Dashboard-Parameter-In-Widget-Embedding/README.md (hunk 1 above)
Comment (naveena912) [reply Scenario Based Samples/Dashboard-Parameter-In-Widget-Embedding/README.md:93]: Resolved the concern and changed the content

File: Scenario Based Samples/Dashboard-Parameter-In-Widget-Embedding/README.md (hunk 2 above)
Comment (naveena912) [reply Scenario Based Samples/Dashboard-Parameter-In-Widget-Embedding/README.md:34]: Added the content and resolved the concerns

**Review by naveena912 — COMMENTED** [review]
Resolved the concerns

File: Scenario Based Samples/Dashboard-Parameter-In-Widget-Embedding/README.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+ ## Configuration
+
+  * Please ensure you have enabled embed authentication on the `embed settings` page. If it is not currently enabled, please refer to the following image or detailed [instructions](https://help.boldbi.com/site-administration/embed-settings/#get-embed-secret-code) to enable it.
+
+    ![Embed Settings](https://github.com/boldbi/aspnet-core-sample/assets/91586758/b3a81978-9eb4-42b2-92bb-d1e2735ab007)
+
+  * To download the `embedConfig.json` file, please follow this [link](https://help.boldbi.com/site-administration/embed-settings/#get-embed-configuration-file) for reference. Additionally, you can refer to the following image for visual guidance.
+
+     ![Embed Settings Download](https://github.com/boldbi/aspnet-core-sample/assets/91586758/d27d4cfc-6a3e-4c34-975e-f5f22dea6172)
```
Comment (vinothkumar1732) [inline Scenario Based Samples/Dashboard-Parameter-In-Widget-Embedding/README.md:26]: Use newly created sample URL as I reported previously.

File: Scenario Based Samples/Dashboard-Parameter-In-Widget-Embedding/README.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+2. By default, the `HR Recruitment Dashboard` does not have [Dashboard Parameters](https://help.boldbi.com/cloud-bi/working-with-data-source/configuring-dashboard-parameters/#configuring-dashboard-parameters) configured. For demonstration, follow these steps to configure the Dashboard Parameter with `Literal Mode` as shown in the screenshot below:
+
+    * Enter the Parameter Name: `Department`
+    * Enter the Parameter Value: `IT`
+
+    ![enter-parameter-name-value](https://github.com/boldbi/samples/assets/92368448/d4041b0a-517a-4c3d-abf7-c05e25615630)
+
+3. After successfully publishing the dashboard with configured dashboard parameters, download the [embedConfig JSON](https://help.boldbi.com/site-administration/embed-settings/#get-embed-configuration-file) file associated with that specific dashboard. 
+
+4. Once configured the embedConfig.JSON file in the designated [location](https://github.com/boldbi/aspnet-core-sample/tree/master/BoldBI.Embed.Sample), then run your application.
```
Comment (vinothkumar1732) [inline Scenario Based Samples/Dashboard-Parameter-In-Widget-Embedding/README.md:83]: Use newly created sample URL as I reported previously.

File: Scenario Based Samples/Dashboard-Parameter-In-Widget-Embedding/README.md (hunk 4 above)
Comment (naveena912) [reply Scenario Based Samples/Dashboard-Parameter-In-Widget-Embedding/README.md]: Changed the URL and resolved the concerns

File: Scenario Based Samples/Dashboard-Parameter-In-Widget-Embedding/README.md (hunk 5 above)
Comment (naveena912) [reply Scenario Based Samples/Dashboard-Parameter-In-Widget-Embedding/README.md]: Changed the URL and resolved the concerns

**Review by naveena912 — COMMENTED** [review]
Resolved the concerns

**Pull request merged**